oriedita/src/main/java/oriedita/editor/action/MouseModeHandler.java

```diff
@@ -3,14 +3,69 @@
 import origami.crease_pattern.element.Point;
 import oriedita.editor.canvas.MouseMode;
 
+import java.awt.event.MouseEvent;
+import java.util.ArrayList;
+import java.util.EnumSet;
+import java.util.List;
+
 public interface MouseModeHandler {
+    enum Feature {
```

Comment:
Any reason you gave this a quite generic name and not something like MouseButton? Got ideas to add other features which are not mouse buttons?

Reply:
At first I was thinking about having the triple-click as a separate feature, which is why I had the generic name. I decided against it after a bit, but forgot to change the name back. But you're right, maybe it can be expanded in the future, one example could be having ctrl+scroll change the polygon edge number of the polygon tool or something